### ISSUE TITLE
What is the current random seed behavoir

### ISSUE THREAD

**Issue by rickstaa** [rickstaa/LAC-TF2-TORCH-translation#30]
Currently, we have the ability for the user to specify two different random seeds to make the training more deterministic:

- The `ENV_SEED`: This is the random seed that is used in the environment. 
- The `RANDOM_SEED`: This is the random seed that is used in the rest of the training code (random number generators, weight initializers, samplers).

This is useful when you want to debug and have the same behavior. This feature is disabled by setting `ENV_SEED` and `RANDOM_SEED' to `None`. 

#### ENV_SEED

If you set the `ENV_SEED` to a number, every time you create the environment  the sequence of states out of the `reset()` and `step()` environment will be the same. Meaning that every time you start the training loop the outputs from the environment are the same when the seed is the same. 

#### RANDOM_SEED

If you set the `RANDOM_SEED` to a number every time you start the training the random numbers that are generated in the training loop are predetermined based on the random seed. This means the following:

- The values that are returned by methods that use a random number generator are predetermined. For example, the batch sample method will return the same Experiences from the replay buffer every time you start the training with the same random seed. 

- The initial weights of the networks are also equal in every training run if the same random seed is set.
 This, however, does not mean that the initial weights between Rollouts (policies) in the same training run are equal. It only means that the initial weights of rollout 1.2.3...x are identical every time you start the training with the same random seed. 

#### Further references
- [numpy random seed method](https://www.sharpsightlabs.com/blog/numpy-random-seed/)
- [gym seed method](https://stackoverflow.com/questions/47331235/how-should-openai-environments-gyms-use-env-seed0)
- [tensorflow random seed method](https://www.tensorflow.org/api_docs/python/tf/random/set_seed)
- [How to get reproducible result in tensorflow](https://stackoverflow.com/questions/38469632/tensorflow-non-repeatable-results)
